packages/calcite-components/src/components/accordion-item/accordion-item.scss

```
@@ -1,210 +1,202 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-accordion-border-color: Specifies the border color of the component
+ * @prop --calcite-accordion-text-color: Specifies the text color of the component
+ * @prop --calcite-accordion-text-color-secondary: Specifies the highlighted text color
+ * @prop --calcite-accordion-text-color-emphasis: Specifies the selected text color
```

It looks like we're close, just missing the closing period on the above props. Also added small copyedits for readability:

```suggestion
 * @prop --calcite-accordion-border-color: Specifies the component's border color.
 * @prop --calcite-accordion-text-color: Specifies the component's text color.
 * @prop --calcite-accordion-text-color-secondary: Specifies the highlighted text color.
 * @prop --calcite-accordion-text-color-emphasis: Specifies the selected text color.
```